packages/lib-user/src/components/UserHome/components/WelcomeModal/WelcomeModal.js

```diff
@@ -0,0 +1,185 @@
+import { Anchor, Box, Button, Layer, Paragraph } from 'grommet'
+import {
+  CloseButton,
+  SpacedText,
+  ZooniverseLogo
+} from '@zooniverse/react-components'
+import { useEffect, useState } from 'react'
+import styled from 'styled-components'
+
+const textColor = { light: 'black', dark: 'white' }
+
+const StyledAnchor = styled(Anchor)`
+  width: max-content;
+  height: max-content;
+  border-radius: 5px;
+  font-size: 1rem;
+  padding: 8px;
+  text-align: center;
+  color: white;
+  font-weight: normal;
+  background-color: ${props => props.theme.global.colors['neutral-1']};
+
+  &:hover {
+    text-decoration: none;
+  }
+`
+
+const StyledDismiss = styled(Button)`
+  width: max-content;
+  height: max-content;
+  border-radius: 5px;
+  border: solid 1px ${props => props.theme.global.colors.brand};
+  font-size: 1rem;
+  padding: 8px;
+  text-align: center;
+  color: black;
```

Comment:
I think this needs a different color (white?) for the dark theme.

Reply:
Good catch! I'll fix that before merging.